src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
```

Comment:
Consider using `Object.keys(robot)` or `Object.entries(robot)` to iterate over the properties of the robot object. This approach avoids iterating over properties in the prototype chain and is generally considered a best practice when you only want to deal with the object's own properties.